src/TikTok/Provider.php

```diff
@@ -28,13 +28,17 @@ class Provider extends AbstractProvider
      */
     protected function getAuthUrl($state)
```

Comment:
Can we get an `@see` docblock on the class with a link to the docs please

Reply:
Done. Added link to the TikTok announcement blog post and to the OAuth docs.